.github/workflows/hive.yaml

```diff
@@ -24,12 +24,18 @@ jobs:
           - simulation: rpc-compat
             name: "Rpc Compat tests"
             run_command: make run-hive SIMULATION=ethereum/rpc-compat TEST_PATTERN="/eth_chainId|eth_getTransactionByBlockHashAndIndex|eth_getTransactionByBlockNumberAndIndex|eth_getCode|eth_getStorageAt|eth_call|eth_getTransactionByHash|eth_getBlockByHash|eth_getBlockByNumber|eth_createAccessList|eth_getBlockTransactionCountByNumber|eth_getBlockTransactionCountByHash|eth_getBlockReceipts|eth_getTransactionReceipt|eth_blobGasPrice|eth_blockNumber|ethGetTransactionCount|debug_getRawHeader|debug_getRawBlock|debug_getRawTransaction|debug_getRawReceipts|eth_estimateGas|eth_getBalance|eth_sendRawTransaction|eth_getProof|eth_getLogs"
-          - simulation: rpc-engine
-            name: "Engine Auth tests"
+          - simulation: rpc-auth
+            name: "Rpc Auth tests"
             run_command: make run-hive SIMULATION=ethereum/rpc-compat TEST_PATTERN="/engine-auth"
           - simulation: discv4
             name: "Devp2p discv4 tests"
-            run_command: make run-hive-debug SIMULATION=devp2p TEST_PATTERN="discv4"
+            run_command: make run-hive SIMULATION=devp2p TEST_PATTERN="discv4"
+          - simulation: engine
+            name: "Engine tests"
+            run_command:  make run-hive SIMULATION=ethereum/engine TEST_PATTERN="/Blob Transactions On Block 1, Cancun Genesis|Blob Transactions On Block 1, Shanghai Genesis|Blob Transaction Ordering, Single Account, Single Blob|Blob Transaction Ordering, Single Account, Dual Blob|Parallel Blob Transactions|ForkchoiceUpdatedV3 Modifies Payload ID on Different Beacon Root|NewPayloadV3 After Cancun|NewPayloadV3 Versioned Hashes|ForkchoiceUpdated Version on Payload Request"
```

Comment:
not sure I understand. Are these not cancun too?

Reply:
There are some tests that have two variants (Paris) and (Cancun), as they are not part of the name we cannot filter the cancun test variants (aka TEST_NAME (Cancun) is not valid), but filtering by `cancun/TEST_NAME` works well. I initially had these tests within the same pattern but it wasn't working in the CI (the cancun/ tests were being ignored), but it started working well once I split the pattern into two.
I also think the split is nice to keep CI times lower